File: src-front/VueApp/ClientApp/src/models/Space.ts
import { BaseModel } from './BaseModel';
import { Page } from './Page';

export class Space extends BaseModel {
  public id!: string;
  public name!: string;
  public slug!: string;
  public description?: string;
  // public owner!: UserLightDTO;
  public pages?: Page[];
}

File: src-back/Note.Core/Entities/Space.cs
using Note.Core.Entities.Base;
using System.Collections.Generic;

namespace Note.Core.Entities
{
    public class Space : Entity
    {
        public string Name { get; set; }
        public string Slug { get; set; }
        public string Description { get; set; }

        public AppUser Owner { get; set; }
        public virtual ICollection<Page> Pages { get; set; }
    }
}
